bgpd/bgp_fsm.h

```diff
@@ -32,7 +32,7 @@
 #define BGP_TIMER_OFF(T)                                                       \
 	do {                                                                   \
 		if (T)                                                         \
-			THREAD_TIMER_OFF(T);                                   \
+			THREAD_OFF(T);                                   \
 	} while (0)
```

Comment:
IMO let's delete this whole macro, it does nothing; the nullity check is in `THREAD_OFF` and also in `thread_cancel` so that's three checks for no reason